css/custom.css

```diff
@@ -0,0 +1,371 @@
+  height: 3rem;
+}
+
+.btn-sml {
+  height: 2rem;
+  padding-bottom: 2rem;
+  border-radius: 40px;
+  border: none;
+}
+
+.upper {
+  background: #f72210;
+  height: 7rem;
+}
+
+.login {
+  color: white;
+}
+
+.bg-warning {
+  padding-top: 0.5rem;
+  border: none;
+  border-radius: 20px;
+  width: 180px;
+  height: 40px;
+}
+
+.title img {
+  height: 5rem;
+  width: auto;
+}
+
+li {
+  list-style: none;
+}
+
+.last {
+  border: none;
+}
+
+.middle input {
+  border: none;
+  text-align: center;
+  width: 4rem;
+}
+
+.nav-link {
+  text-align: center;
+  padding-top: 1rem;
+  margin: 0;
+}
+
+.lower a:hover {
+  color: red;
+}
+
+.pos {
+  background-color: white;
+  width: auto;
+  bottom: 0;
+  left: 0;
+}
+
+.pos h6 {
+  background-color: lightcyan;
+  color: black;
+}
+
+.pos h6,
+.pos h5 {
+  margin: 0;
+}
+
+.top_img {
+  padding-left: 1rem;
+}
+
+.debate img {
+  height: 4rem;
+}
+
+.debate_head {
+  border-bottom: 2px groove lightgray;
+}
+
+.vs {
+  width: 50px;
+  height: 50px;
+  top: 4.5%;
+  left: 45%;
+}
+
+.mag img {
+  width: 2rem;
+  height: 2rem;
+}
+
+.mag_hov:hover {
+  background-color: lightgray;
+}
+
+/** Sports **/
+
+.small-img {
+  width: 100px;
+  float: left;
+  margin: 10px;
+}
+
+.dif {
+  color: gray;
+}
+
+.dif span {
+  font-size: x-small;
+}
+
+.col-blue {
+  color: blue;
+}
+
+.dif2 span {
+  color: black;
+  font-size: small;
+}
+
+/* Subscribe 5 */
+
+.article-cont {
+  width: 100%;
+  padding: 25px;
+  background-color: rgb(244, 245, 246);
+}
+
+.input-cont [type=text],
+[type=submit] {
+  width: 100%;
+  padding: 10px;
+  margin: 10px 0;
+  border: solid 1px black;
+  box-sizing: border-box;
+  display: inline-block;
+}
+
+.input-cont [type=submit],
+.text-cont [type=submit] {
+  background-color: rgb(246, 22, 2);
+}
+
+.text-cont input {
+  width: 100px;
+  border: none;
+  padding: 0;
+}
+
+/** Subscribe **/
+
+.container-sub {
+  background-color: rgb(244, 245, 246);
+  padding: 10px;
+}
+
+.news-letter h6 {
+  color: rgb(247, 34, 16);
+}
+
+.newsletter {
+  padding: 0;
+  border: solid 1px lightgray;
+}
+
+.cont-letter input[type="text"],
+[type="submit"] {
+  width: 100%;
+  display: inline-block;
+  box-sizing: border-box;
+}
+
+.submit [type="submit"] {
+  background-color: rgb(247, 34, 16);
+  border: none;
+  margin-top: 7px;
+}
+
+.letter-img {
+  float: left;
+}
+
+.boxing {
+  background-color: rgb(243, 244, 245);
+  width: auto;
+}
+
+/** Footer **/
+
+.f-end {
+  align-items: flex-end;
+}
+
+.mr-5px {
+  margin-right: 5px;
+}
+
+.sticky {
+  position: sticky;
+  top: 0;
+  left: 0;
+  align-self: end;
+}
+
+.news_letter {
+  height: 300px;
+  background: url(https://d.newsweek.com/en/full/1541302/starting5-homepage.png?w=397&h=265&q=90&f=742bce9a6f72dd8f19ba2e7fd5db06e5) center center no-repeat;
+  background-size: cover;
+}
+
+.text-color {
+  background-color: rgb(244, 245, 246);
+}
+
+.text-box {
+  height: 300px;
+  width: 100%;
+}
+
+@media only screen and (max-width: 767px) {
```

Comment:
Since we are following the mobile-first approach, we will stick to the `min-width` only approach, and the recommended breakpoint should be 768px.